src/cljs/athens/core.cljs

```diff
@@ -34,7 +34,7 @@
   []
   (rf/clear-subscription-cache!)
   (router/init-routes!)
-  (r-dom/render [views/main-panel]
+  (r-dom/render [views/view]
```

Comment:
It feels like `main-panel` was a better name.

Reply:
renamed to `main`. I just didn't understand what a `panel` as opposed to just a page.